lib/commonBindings.js

```diff
@@ -55,7 +55,13 @@ function parseMessage(message) {
 
     try {
         stringMessage = message.toString();
-        parsedMessage = JSON.parse(stringMessage);
+        try {
+            parsedMessage = JSON.parse(stringMessage);
+        } catch (e) {
+            stringMessage = message.toString('hex');
```

Comment:
parsedMessage =  message.toString('hex');

Reply:
done in 6a6d2f58cc6d5fa9bd456dd7dac4655d901e7a76